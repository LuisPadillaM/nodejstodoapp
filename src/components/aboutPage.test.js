import React from 'react';
import renderer from 'react-test-renderer';
import AboutPage from "./aboutPage";

test('About page render test', () => {
  const component = renderer.create(
    <AboutPage/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
