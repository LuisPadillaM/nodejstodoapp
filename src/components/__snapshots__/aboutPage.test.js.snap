// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page render test 1`] = `
<h3>
  About
</h3>
`;
